tools/apidiff/cmd/diff.go

```diff
@@ -23,12 +23,12 @@ import (
 )
 
 var diffCmd = &cobra.Command{
-	Use:   "diff <base export filepath> <target export filepath>",
+	Use:   "diff <base export filepath> <target export filepath> <release tag version> <release marker>",
```

Comment:
Add explanation for new params.

Reply:
Remove "release marker" per our discussion.

Reply:
Done and done.